soc/nordic/Kconfig

```diff
@@ -73,6 +73,13 @@ config NRF_ACL_FLASH_REGION_SIZE
 	help
 	  FLASH region size for the NRF_ACL peripheral.
 
+config NRF_RRAM_REGION_SIZE_GRANULARITY
```

Comment:
This value applies to the RRAM peripheral, not to all Nordic SoCs. I think it should be placed in https://github.com/nrfconnect/sdk-zephyr/blob/main/drivers/flash/Kconfig.nrf_rram instead.